springboard/config.py

```diff
@@ -4,4 +4,5 @@ def includeme(config):
     config.add_route('category', '/category/{uuid}/')
     config.add_route('page', '/page/{uuid}/')
     config.add_route('flat_page', '/{slug}/')
+    config.add_route('api_notify', '/api/notify/', request_method='POST')
```

Comment:
I take it our api's stuff that we currently have in `unicore-cms` haven't been ported yet?